southpark-search/get_data.sh

```diff
@@ -12,4 +12,4 @@ fi
 
 mkdir -p ${TEST_DATA_DIR}
 git clone https://github.com/BobAdamsEE/SouthParkData.git $TEST_DATA_DIR
-python prepare_data.py ${TEST_DATA_DIR}
\ No newline at end of file
+python3 prepare_data.py ${TEST_DATA_DIR}
```

Comment:
Let's stick to `python` without version

Reply:
Oh yes, I changed it for testing, thank you for reminding